pubspec.yaml

```diff
@@ -8,23 +8,23 @@ environment:
   sdk: ">3.0.0"
 
 dependencies:
-  android_id: ^0.1.3+1
-  connectivity_plus: ^3.0.3
-  device_info_plus: ^8.0.0
+  android_id: ^0.3.6
+  connectivity_plus: ^5.0.2
+  device_info_plus: ^8.2.2
```

Comment:
There are newer versions of `device_info_plus` (`9.1.1`) and `package_info_plus` (`5.0.1`), but explorer-ui depends on `wakelock` `0.6.2` and it requires the older dependencies.

Reply:
`wakelock` has been superceded by [`wakelock_plus`](https://pub.dev/packages/wakelock_plus).